oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/ProfileTaskCreationResult.java

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package org.apache.skywalking.oap.server.core.profile.entity;
+package org.apache.skywalking.oap.server.core.query.entity;
```

Comment:
I think this is mischange?

Reply:
> I think this is mischange?

The GraphQL resolver will generate some classes for the query entities, which breaks the Jacoco agent, we have to exclude these classes in Jacoco, though I can have another rules to exclude this particular class (`ProfileTaskCreationResult`), but I think it belongs package `org.apache.skywalking.oap.server.core.query.entity`, as other similar query entities

Reply:
Good to me.